pkg/graphql/execution_engine_v2_test.go

```diff
@@ -887,6 +887,68 @@ func TestExecutionEngineV2_Execute(t *testing.T) {
 		expectedResponse: `{"data":{"heroes":["Human","Droid"]}}`,
 	}))
 
+	t.Run("execute operation with null and omitted input variables", runWithoutError(ExecutionEngineV2TestCase{
```

Comment:
Can you also add a test where we omit `{"height":null}` ? Or does this test already exist?

Reply:
we omit `names` in this test, and `height` is set to `null` so it remains